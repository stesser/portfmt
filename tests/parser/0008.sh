echo ".endfor" | ${PORTFMT} -d 2>&1 | diff -L "0008.expected" -L "0008.actual" -u 0008.expected -
