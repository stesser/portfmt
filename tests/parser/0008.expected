portfmt: line 2: error building AST: could not find matching .for for .endfor on line 1-2
